README.md

```diff
@@ -43,11 +43,16 @@ By performing a [GaAttachTx](https://aeternity.com/protocol/generalized_accounts
     - params
         - `hash` the tx-hash of the meta transaction that should be authorized
     - can be called to confirm the currently proposed tx
-    - the signature is checked against the stored hash
+- `refuse(hash)`
+    - params
+        - `hash` the tx-hash of the meta transaction that should be authorized
+    - can be called to refuse the currently proposed tx
 - `revoke(hash)`
     - params
         - `hash` the tx-hash of the meta transaction that should be authorized
     - can be called to explicitely revoke the currently proposed tx
+        - no check if the caller is the proposer of the tx
```

Comment:
```suggestion
        - now checks if the caller is the proposer of the tx
```

Reply:
yeah that was a bit of an unfortunate statement 😅

should have written sth. like "passes if the caller is proposer of the tx without additional check" or so 🤔

Reply:
I consider this resolved now :)